runtime/rococo/src/lib.rs

```diff
@@ -205,6 +209,11 @@ construct_runtime! {
 
 		// Propose parachain pallet.
 		ProposeParachain: propose_parachain::{Module, Call, Storage, Event<T>},
+
+		// Bridges support.
+		Mmr: pallet_mmr::{Module, Call, Storage},
+		MmrLeaf: mmr::{Module},
+		Beefy: pallet_beefy::{Module, Config<T>},
```

Comment:
Shouldn't beefy `Storage` be made available so we can do a state query for the current authority set?

Reply:
When we tried querying using Polkadot-JS, `api.query.beefy.authorities()` API wasn't available, and I suspect its due to the issue in my above comment.